client/components/aquarium-form.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+export default class AquariumForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      size: '',
+      file: ''
+    };
+    this.fileInputRef = React.createRef();
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.renderPreview = this.renderPreview.bind(this);
+    this.setSize = this.setSize.bind(this);
+    this.setName = this.setName.bind(this);
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData();
+    formData.append('name', this.state.name);
+    formData.append('size', this.state.size);
+    formData.append('image', this.fileInputRef.current.files[0]);
+
+    fetch('/api/aquariums', {
+      method: 'POST',
+      body: formData
+    })
+      .then(response => response.json())
+      .then(resBody => {
+        this.setState({
+          name: '',
+          size: '',
+          file: ''
+        });
+        this.fileInputRef.current.value = null;
+      })
+      .catch(error => {
+        console.error('Error:', error);
+      });
+    const form = document.querySelector('#aquarium-form');
+    form.reset();
```

Comment:
In react, we try to never query the document like this. Your form should reset by the setState you did.